pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -39,6 +40,9 @@ const (
 
 	// See https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2570-memory-qos
 	DefaultMemoryThrottlingFactor = 0.8
+
+	DefaultKubeletRunDir                  = "/var/lib/kubelet"
```

Comment:
https://github.com/kubernetes/kubernetes/blob/cde45fb161c5a4bfa7cfe45dfd814f6cc95433f7/cmd/kubeadm/app/constants/constants.go#L272

There is an constant already.

Reply:
I did not want to import to apis from a cmd package, both had different purposes.